src/router.ts

```diff
@@ -227,5 +240,10 @@ export default new Router({
             component: SignMessageSuccess,
             name: `${RequestType.SIGN_MESSAGE}-success`,
         },
+        {
+            path: `/${RequestType.SIGN_MESSAGE}/ledger`,
+            component: ErrorHandlerUnsupportedLedger,
+            // not specifying a name here to not trigger automatic routing to this view in RpcApi.ts
```

Comment:
Why is the name omitted here but in the 2 other cases it is not?

Reply:
Because for export and change password, the call should always reject.
For sign message, the user is given the option to chose another account unless the `forceSender` flag (not implemented yet) is set in which case he would be forwarded to the error page.